src/app.js

```diff
@@ -1,16 +1,29 @@
-const express = require("express");
+const LogStream = require("./myStream");
 const bodyParser = require("body-parser");
+const errorHandler = require("./error-handler");
+const express = require("express");
+const morgan = require("morgan");
 const repositoryCotroller = require("./controllers/repository");
+const userCotroller = require("./controllers/user");
+const { HttpNotFound } = require("./utils/errors");
 
+const myStream = new LogStream();
 const app = express();
 
+app.use(morgan("dev", { stream: myStream }));
```

Comment:
```suggestion
app.use(morgan("dev", { stream: new LogStream() }));
```

Se esse stream não eh usado em outro lugar, talvez valesse evitar a variavel. Também legal evitar variaveis que começam com `my` já que geralmente n trazem um significado muito util.